sources/xfscli/src/bin/xfs_info.rs

```diff
@@ -0,0 +1,61 @@
+use std::process::exit;
+use std::process::Command;
+use std::str;
+
+use argh::FromArgs;
+
+/// display XFS filesystem geometry information
+#[derive(FromArgs)]
+struct Args {
+    /// print the version
+    #[argh(switch, short = 'V')]
+    version: bool,
+    /// mount-point | block-device | file-image
+    #[argh(positional)]
+    target: Option<String>,
+}
+
+static XFS_SPACEMAN: &str = "/usr/sbin/xfs_spaceman";
+static FINDMNT: &str = "findmnt";
+static XFS_DB: &str = "/usr/sbin/xfs_db";
+
+fn main() {
+    let args: Args = argh::from_env();
+
+    // Return the version for arg version as true
+    if args.version {
+        let xfs_spaceman_result = Command::new(XFS_SPACEMAN)
+            .args(["-p", "xfs_info", "-V"])
+            .output()
+            .expect("Couldn't run xfs_spaceman for version info");
+        print!("{}", String::from_utf8(xfs_spaceman_result.stdout).unwrap());
+        exit(0);
+    }
+
+    if let Some(target) = &args.target {
+        // Check if target is mounted
+        let mountpt = Command::new(FINDMNT)
+            .args(["-t", "xfs", "-f", "-n", "-o", "TARGET", target.as_str()])
+            .output()
+            .expect("Couldn't find the mount point");
```

Comment:
nit: Can we get this information from https://docs.rs/mnt/latest/mnt/ or another library? I'd like to avoid using `Command` to remove specific dependencies on this command returning input and accepting parameters exactly as they do right now. Not that `findmnt` is likely to change, its more of a general stance.